tests/03_regression/atmospheres/test_rpv_afgl1986_continental.py

```diff
@@ -35,6 +36,8 @@ def test_rpv_afgl1986_continental_brfpp(
     )
     result = eradiate.run(exp, spp=10000)
 
+    logger.info(result._repr_html_(), html=True)
```

Comment:
That's nice!

Reply:
Yes! Suggestions are welcome in order to add additional charts and visual constructs to the report. I believe this should be done per test, case by case, and not systematically. We could also encourage the design of meaningful debug charts when developers submit new tests